src/RequestHandler/RequestManager.js

```diff
@@ -0,0 +1,40 @@
+const {HandlerFactory} = require('./HandlerFactory');
+const supportedProtocols = ['mqtt'];
+function RequestManager () {
+  const handlersList = {};
+
+  async function createReqHandler(dataFromParser) {
+    for (const [serverName,serverData] of Object.entries(dataFromParser.servers)) {
+      const handlerInstance = await HandlerFactory(serverData , dataFromParser.PublishOperations).catch((err) => {
+        console.log(err.message);
+        return 'notHealthy';
```

Comment:
You will have to revise this implementation, you cannot return inside a `catch` statement like so. You return `notHealthy` to the catch callback, not the `createReqHandler` function.